### PR TITLE
Specter update to v0.7.2

### PR DESCRIPTION
**Pull request by openoms**
installs and runs well on RaspiBlitz v1.6 

Related: https://github.com/rootzoll/raspiblitz/issues/1411

### CODE REVIEW

**Comment by openoms** [comment]
To update manually from Specter 0.5.5 (in raspiblitz v1.6):
```
#uninstall old version
/config.scripts/bonus.cryptoadvance-specter.sh off
# install the missing dependency
sudo apt install -y libffi-dev
# change script to use the latest version
sed -i "s#cryptoadvance.specter==0.5.5#cryptoadvance.specter==0.7.2#g"  /home/admin/config.scripts/bonus.cryptoadvance-specter.sh
```
and install from SERVICES

**Comment by k9ert** [comment]
Sorry to have not spotted that earlier. Here one have to change the version explicitely in the pip-command. Why have that been introduced in this commit at all @rootzoll  :
https://github.com/rootzoll/raspiblitz/commit/a5864f71243e3f8eb2222814dcb349edf6584292
The beauty and the difference to mynode was that the latest version got installed. If one wants to pin it, he can still add the version but the default should imho be to install the latest version, no?
Specter is doing very quick releases and is fixing a lot o bugs rolling forward. So i can't see many arguments pinning the version.

**Comment by ChuckNorrison** [comment]
Stick to a specific revision is a basic question of usage. For the RaspiBlitz project, i like it "the sticky" way, for a more safe first time user experience and make room to grow with the system and get some ideas how it works. It should not be hard to override the sticky revisions manually after basic knowledge of the setup without much untested things in to get interrupted on this way.

But great idea to get some more comfort on this topic. This could be extended via the Patch menu to take control of the repos in use

**Comment by openoms** [comment]
@k9ert our aim is to know exactly what is running on each raspiblitz version and if someone made manual changes can always fall back to a reinstall or tested sdcard image.

Specter can make it to the UPDATE menu as well if we find a reliable way to.do it. Can be simple pip upgrade. That would add an update option to the script too.

**Comment by ChuckNorrison** [comment]
Hey, i tested the upgrade successful with this:

> sudo -u bitcoin /home/bitcoin/.specter/.env/bin/python3 -m pip install --upgrade cryptoadvance.specter==0.7.2
> sudo systemctl restart cryptoadvance-specter

Great update here

// fixed

**Comment by dennisreimann** [comment]
Can confirm this works too. Used the upgrade command proposed by @ChuckNorrison above, though for me it was `sudo systemctl restart cryptoadvance-specter` to restart the service. (dash instead of period in the service name)

File: home.admin/config.scripts/bonus.cryptoadvance-specter.sh [hunk 1]
```diff
@@ -287,5 +287,15 @@ if [ "$1" = "0" ] || [ "$1" = "off" ]; then
   exit 0
 fi
 
+# update
+if [ "$1" = "update" ]; then
+  echo "#    --> UPDATING Cryptoadvance Specter"
+  sudo -u bitcoin /home/bitcoin/.specter/.env/bin/python3 -m pip install --upgrade cryptoadvance.specter
```
Comment (openoms) [inline home.admin/config.scripts/bonus.cryptoadvance-specter.sh:293]: Added the option to upgrade to the latest from https://pypi.org/project/cryptoadvance.specter/#history once the pinned Specter version is installed.

**Pull request merged**